.github/workflows/crio.yml

```diff
@@ -58,17 +58,6 @@ jobs:
         run: |
           curl https://raw.githubusercontent.com/cri-o/cri-o/master/scripts/get | sudo bash
 
-      - name: Install CRI-O latest version
-        if: ${{matrix.version == 'v1.20.0'}}
-        run: |
-          DIR=crio-${{matrix.version}}
-          TARBALL=$DIR.tar.gz
-          RELEASE=gs://k8s-conform-cri-o/artifacts/$TARBALL
-          gsutil cp $RELEASE .
-          tar xf $TARBALL
-          pushd $DIR || exit
-          sudo -E PATH=$PATH make install
```

Comment:
I'll readd this with the CRI-O v1.22.0 release.

Reply:
So the test added here would fail for cri-o v1.20.0?

Reply:
We had a regression with those new tests which get fixed with CRI-O v1.22.0. The release is still pending so I can re-add them when it's done.